### ISSUE TITLE
[LiteLLM Client] Add new models via UI 

### ISSUE THREAD

**Issue by krrishdholakia** [BerriAI/litellm#295]
**Describe the solution you'd like**
Creating a ticket to track adding a new model via the UI on LiteLLM Client. If you're currently using admin.litellm.ai and trying to add new models, comment below explaining you're use-case and how you'd expect it to work? 

Open Questions 
- How do you pick which model to route traffic to?

**Comment by shuther** [comment]
is it possible to add a connectionto  https://app.edenai.run/ - https://www.edenai.co/ ?

**Comment by kno2gether** [comment]
Checkout this simple UI. I m not a much of UI Person. Used AppSmith and used LiteLLM API to create this simple UI
https://github.com/kno2gether/LitellmProxyUI

Tutorial here - https://www.youtube.com/watch?v=-Wo025I-_I4

**Comment by ghost** [comment]
this is super cool @kno2gether

**Comment by ghost** [comment]
@kno2gether looking at your config.yaml, I'm curious why did you set it up this way? 

<img width="669" alt="Screenshot 2023-12-21 at 9 43 02 PM" src="https://github.com/BerriAI/litellm/assets/17561003/edf75420-24a7-404e-b309-aef693e5e568">


For example, to call openrouter you can just do 

```yaml
- model_name: mistral-7b-instruct
  litellm_params: 
      model: openrouter/mistralai/mistral-7b-instruct
      api_key: <your-api-key> 
```

Were there any docs you looked at for this? Want to make it better for you.

**Comment by kno2gether** [comment]
Yeah ! ofcourse, I could have used the other approach (more sort of module/modelname) but I preferred to use it for the tutorial video so that an AI NewBie could understand how its working under the hood. Ofcourse, in a more production setup, I'd not keep the api keys in the config yaml as well.

Also, regarding docs, yes I referred the config example specified here https://docs.litellm.ai/docs/proxy/configs.

If there are plans to drop the usage (As I did earlier), would be worth to update the docs reference.
